src/styles/main.scss

```diff
@@ -1,3 +1,78 @@
+$cellSize: 100px;
+$borderColor: #000;
+$backgroundColor: #eee;
+$textColor: #000;
+$hover: pink;
+$numberFont: "Arial";
+$numberSize: 30px;
+$gap: 1px;
+$padding: 10px;
+$daysInMonth: 31;
+$calendarWidth: calc(7 * #{$cellSize} + 8 * #{$gap} + 2 * #{$padding});
+$monthLength: 28, 29, 30, 31;
+$startDay: (
+  "mon": 0,
+  "tue": $cellSize + $gap,
+  "wed": 2*($cellSize + $gap),
+  "thu": 3*($cellSize + $gap),
+  "fri": 4*($cellSize + $gap),
+  "sat": 5*($cellSize + $gap),
+  "sun": 6*($cellSize + $gap)
+);
```

Comment:
You are using variables - which is good 👍 

However, you have many variables, so this part look bit messy

Please use line breaks to separate some variables, based on types (say colors, gaps/sizes/ logic, etc)
Also please be consistent naming colors
If you have `$borderColor: #000;` then you need to rename `pink` to `hoverColor`, `cellHoverColor`, etc